restapi/src/main/java/io/stargate/web/docsapi/service/QueryExecutor.java

```diff
@@ -54,6 +54,12 @@ public QueryExecutor(DataStore dataStore) {
       int pageSize,
       ByteBuffer pagingState,
       ExecutionContext context) {
+    if (pageSize <= 0) {
```

Comment:
aren't you using the page size 0 when populating the docs?

Reply:
I think default is 1000

```java
  public static final Integer SEARCH_PAGE_SIZE =
      Integer.getInteger("stargate.document_search_page_size", 1000);
```

Reply:
I think I replaced all places where `0` was used with "search page size". Please let me know if I missed something.